src/components/BooksUI/Booklist.js

```diff
@@ -1,8 +1,15 @@
-import { useSelector } from 'react-redux';
+import { useSelector, useDispatch } from 'react-redux';
+import { useEffect } from 'react';
 import Bookinfo from './Bookinfo';
+import { getBooks } from '../../redux/books/api';
 
 const Booklist = (() => {
-  const state = useSelector((state) => state.booksReducer);
+  const state = useSelector((state) => state.booksReducer.books);
+
+  const dispatch = useDispatch();
+  useEffect(() => {
+    dispatch(getBooks());
+  }, [dispatch]);
```

Comment:
- [ ] Good use of `useEffect` hook to make request to the server 👍 💯 
- [ ] The user is now able to see added books after the page is refreshed. 👍